sync/_databricks.py

```diff
@@ -1725,6 +1725,25 @@ def _dbfs_directory_has_all_rollover_logs(contents: dict, run_end_time_millis: f
     )
 
 
+def _dbfs_any_file_has_zero_size(dbfs_contents: Dict):
```

Comment:
Super nitpicky but perhaps you also wanted a return type hint `-> bool | None`

Reply:
good catch. Type hints are important, definitely not a nit :)